index.html

```diff
@@ -28,6 +29,69 @@ <h3>I'm a Software Developer.</h3>
                 <li><a href="https://twitter.com/Luffytaro_san22" target="_blank" rel="noopener"><i class="fa-brands fa-twitter"></i></a></li>
             </ul>
         </section>
+        <section id="works">
+            <h2>My Recent Works</h2>
+            <img src="./Normal Button/Tertiary/Icons/Img Placeholder.png" alt="Works-placeholder">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selections of privately personalized reads; no accounts or sign-ups required. Take your time and see the cool works I've done.</p>
+            <ul class="languages">
+                <li>CSS</li>
+                <li>HTML</li>
+                <li>Javascript</li>
+            </ul>
+            <a href=""><button type="button">See project</button></a>
+            <div class="works-container">
+                <div class="works">
+                    <h2>Calculator</h2>
+                    <p>A simple calculator that can do the basics operations.</p>
+                    <ul class="lw languages">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <a href="https://luffytaro22.github.io/Calculator/" target="_blank" rel="noopener"><button type="button">See project</button></a>
+                </div>
+                <div class="works">
+                    <h2>Etch-A-Sketch</h2>
+                    <p>A recreation of the Etch-A-Sketch interactive dashboard where you can select the quantity of spaces and the colors.</p>
+                    <ul class="lw languages">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <a href="https://luffytaro22.github.io/Etch-A-Sketch/" target="_blank" rel="noopener"><button type="button">See project</button></a>
+                </div>
+            </div>
+        </section>
+        <section id="about-myself">
+            <h1>About me</h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
```

Comment:
- [ ] I would suggest you to create a `<div>` tag to  `<h1>`tag and your `<p>`tag so as to make your code more readable and semantic.
- [ ] Kindly update all the social media icon   exactly like the figma template